### PR TITLE
#176 Hide profile and setting tab

### PR DESCRIPTION
**Pull request by jingtechy**
## Description

The PR hides the Profile and Settings tabs and only displays the Logout icon.

## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [x] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Related Tickets & Documents

[PIT-176](https://das-ph-inventory-tracker.atlassian.net/browse/PIT-176)

## QA Instructions, Screenshots, Recordings

It has been tested locally, and the screenshot has been attached to the user story.

### CODE REVIEW

**Review by pingchenchan — APPROVED** [review]

**Review by aitzeng — APPROVED** [review]

File: src/layout/MainLayout/Header/HeaderContent/Profile/index.tsx [hunk 1]
```diff
@@ -1,4 +1,6 @@
-import { ReactNode, useContext, useEffect, useRef, useState } from 'react';
+import {
+//     ReactNode,
```
Comment (bospoort) [inline src/layout/MainLayout/Header/HeaderContent/Profile/index.tsx:2]: Can we remove the commented out line?

File: src/layout/MainLayout/Header/HeaderContent/Profile/index.tsx [hunk 2]
```diff
@@ -8,65 +10,67 @@ import {
   CardContent,
   ClickAwayListener,
   Grid,
-//   IconButton,
+  IconButton,
   Paper,
   Popper,
   Stack,
-  Tab,
+//   Tab,
```
Comment (bospoort) [inline src/layout/MainLayout/Header/HeaderContent/Profile/index.tsx:17]: remove?

File: src/layout/MainLayout/Header/HeaderContent/Profile/index.tsx [hunk 3]
```diff
@@ -8,65 +10,67 @@ import {
...
   Tabs,
   Typography,
 } from '@mui/material';
-import { Direction, useTheme } from '@mui/material/styles';
+import {
+//     Direction,
```
Comment (bospoort) [inline src/layout/MainLayout/Header/HeaderContent/Profile/index.tsx:22]: remove?

**Review by bospoort — CHANGES_REQUESTED** [review]
Can we remove the commented out code? We can always get it back from github if needed.

**Review by bospoort — APPROVED** [review]

**Pull request merged**